### ISSUE TITLE
java.lang.RuntimeException: Error creating extended parser class: null

### ISSUE THREAD

**Issue by aesteve** [maiflai/gradle-scalatest#54]
Hello, and thank you for your work on this plugin.

I tried both 0.16 and 0.17 versions (Gradle 4.1) :

``` 
// ...
    ext.scala_major = '2.11'
    dependencies {
        classpath 'com.github.maiflai:gradle-scalatest:0.17'
    }
// ...
apply plugin: 'com.github.maiflai.scalatest'
// ...
    testCompile "org.scalatest:scalatest_$scala_major:3.0.1"
    testRuntime 'org.pegdown:pegdown:1.4.2'
// ...

```

And a single simple test : 

```scala
class RichDateSpec extends FlatSpec with Matchers {

  private val fmt = new SimpleDateFormat("dd/MM/yyyy hh:mm:ss")
  private val TWO_AM = LocalTime.of(2, 0)
  private val oneHourBefore = LocalDateTime.parse("2005-01-04T01:00")
  private val oneHourAfter = LocalDateTime.parse("2005-01-04T03:00")
  private val oneHourOneSecondBefore = LocalDateTime.parse("2005-01-04T00:59:59")

  "timeTo invoked on a past time" should " map to tomorrow" in {
    oneHourAfter.timeTo(TWO_AM) should be(23 hours)
  }

  "timeTo invoked on a future time" should " map to today" in {
    oneHourBefore.timeTo(TWO_AM) should be(1 hour)
  }

  "timeTo " should "take seconds into account" in {
    oneHourOneSecondBefore.timeTo(TWO_AM) should be((60 minutes) + (1 second))
  }

}
```

All tests run perfectly fine in Idea.

But when invoking `gradle test` I am facing this error : 

```
java.lang.RuntimeException: Error creating extended parser class: null
        at org.parboiled.Parboiled.createParser(Parboiled.java:58)
        at org.pegdown.PegDownProcessor.<init>(PegDownProcessor.java:92)
        at org.pegdown.PegDownProcessor.<init>(PegDownProcessor.java:71)
        at org.pegdown.PegDownProcessor.<init>(PegDownProcessor.java:52)
        at org.pegdown.PegDownProcessor.<init>(PegDownProcessor.java:45)
        at org.scalatest.tools.HtmlReporter.<init>(HtmlReporter.scala:117)
        at org.scalatest.tools.ReporterFactory.createHtmlReporter(ReporterFactory.scala:188)
        at org.scalatest.tools.ReporterFactory.getReporterFromConfiguration(ReporterFactory.scala:239)
        at org.scalatest.tools.ReporterFactory$$anonfun$createReportersFromConfigurations$1.apply(ReporterFactory.scala:248)
        at org.scalatest.tools.ReporterFactory$$anonfun$createReportersFromConfigurations$1.apply(ReporterFactory.scala:247)
        at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:234)
        at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:234)
        at scala.collection.Iterator$class.foreach(Iterator.scala:891)
        at scala.collection.AbstractIterator.foreach(Iterator.scala:1334)
        at scala.collection.IterableLike$class.foreach(IterableLike.scala:72)
        at org.scalatest.tools.ReporterConfigurations.foreach(ReporterConfiguration.scala:42)
        at scala.collection.TraversableLike$class.map(TraversableLike.scala:234)
        at org.scalatest.tools.ReporterConfigurations.map(ReporterConfiguration.scala:42)
        at org.scalatest.tools.ReporterFactory.createReportersFromConfigurations(ReporterFactory.scala:247)
        at org.scalatest.tools.ReporterFactory.getDispatchReporter(ReporterFactory.scala:251)
        at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1498)
        at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:1010)
        at org.scalatest.tools.Runner$.main(Runner.scala:827)
        at org.scalatest.tools.Runner.main(Runner.scala)
Caused by: java.lang.IllegalStateException
        at org.objectweb.asm.tree.ClassNode.<init>(ClassNode.java:197)
        at org.parboiled.transform.ParserClassNode.<init>(ParserClassNode.java:41)
        at org.parboiled.transform.ParserTransformer.extendParserClass(ParserTransformer.java:43)
        at org.parboiled.transform.ParserTransformer.transformParser(ParserTransformer.java:38)
        at org.parboiled.Parboiled.createParser(Parboiled.java:54)
        ... 23 more
```

Am I missing something ? Some configuration ? Not sure where I should look at unfortunately.

Thanks for your help.

**Comment by aesteve** [comment]
Answering myself for further reference if anyone's in trouble some day.

https://github.com/Swagger2Markup/swagger2markup/issues/204 this helped a lot.

Forcing pegdown version 1.6.0 solved the issue for me.

Thanks again for your work and sorry for the "noise".

**Comment by maiflai** [comment]
No problem, thanks for the info.

**Comment by wasowski** [comment]
I have hit another instance of this bug, which apparently is not fixed by forcing pegdown to 1.6.0.  But pegdown seems to be used only for HTML reports.  I do not need HTML reports.  So an alternative workaround, for all those stuck is to disable them by adding the following to your build.gradle:

```
test.reports.html.enabled = false
```
